uint8 game_type
uint8 state
uint8 first_half
uint8 kick_off_team
uint8 secondary_state
uint8 secondary_state_team
uint8 secondary_state_info
uint8 drop_in_team
uint16 drop_in_time
uint16 secs_remaining
uint16 secondary_time

uint8   score
uint8   penalty_shot
uint16  single_shots

uint8   opp_score
uint8   opp_penalty_shot
uint16  opp_single_shots

uint8 a_penalty
uint8 a_secs_till_unpenalised
uint8 a_yellow_card_count
uint8 a_red_card_count

uint8 b_penalty
uint8 b_secs_till_unpenalised
uint8 b_yellow_card_count
uint8 b_red_card_count

uint8 c_penalty
uint8 c_secs_till_unpenalised
uint8 c_yellow_card_count
uint8 c_red_card_count

uint8 d_penalty
uint8 d_secs_till_unpenalised
uint8 d_yellow_card_count
uint8 d_red_card_count

uint8 opp_a_penalty
uint8 opp_b_penalty
uint8 opp_c_penalty
uint8 opp_d_penalty

bool    is_kick_off
bool    is_ready
bool    is_set
bool    is_start
bool    is_end